listenbrainz/webserver/static/js/src/recommendations/RecommendationCard.tsx

```diff
@@ -0,0 +1,113 @@
+
+export type RecommendationCardProps = {
+  recommendation: Recommendation;
+  playRecommendation: (recommendation: Recommendation) => void;
+  className?: string;
+  isCurrentUser: Boolean;
+  currentUser?: ListenBrainzUser;
+  newAlert: (
+    alertType: AlertType,
+    title: string,
+    message: string | JSX.Element
+  ) => void;
+};
+
+export default class RecommendationCard extends React.Component<
+  RecommendationCardProps
+> {
+  playRecommendation: (recommendation: Recommendation) => void;
+
+  constructor(props: RecommendationCardProps) {
+    super(props);
+
+    this.playRecommendation = props.playRecommendation.bind(
+      this,
+      props.recommendation
+    );
+  }
+
+  render() {
+    const { recommendation, className } = this.props;
+
+    return (
+      <Card
+        onDoubleClick={this.playRecommendation}
+        className={`recommendation-card row ${className}`}
+      >
+        <div className="col-xs-9">
+          <MediaQuery minWidth={768}>
+            <div className="col-xs-9">
+              <div className="track-details">
+                <p title={recommendation.track_metadata.track_name}>
+                  {getTrackLink(recommendation)}
+                </p>
+                <p>
+                  <small
+                    className="text-muted"
+                    title={recommendation.track_metadata.artist_name}
+                  >
+                    {getArtistLink(recommendation)}
+                  </small>
+                </p>
+              </div>
+            </div>
+          </MediaQuery>
+          <MediaQuery maxWidth={767}>
+            <div className="col-xs-12">
+              <div className="track-details">
+                <p title={recommendation.track_metadata.track_name}>
+                  {getTrackLink(recommendation)}
+                </p>
+                <p>
+                  <small
+                    className="text-muted"
+                    title={recommendation.track_metadata.artist_name}
+                  >
+                    {getArtistLink(recommendation)}
+                  </small>
+                </p>
+              </div>
+            </div>
+          </MediaQuery>
+        </div>
+        <div className="col-xs-3 text-center">
+          <div className="recommendation-controls">
+            <>
+              <RecommendationControl
+                icon={faAngry}
+                title="I never want to hear this again!"
+              />
+              <RecommendationControl
+                icon={faSadCry}
+                title="I don't like this!"
+              />
+              <RecommendationControl
+                icon={faMeh}
+                title="This is a bad recommendation!"
+              />
+              <RecommendationControl icon={faSmileBeam} title="I like this!" />
+              <RecommendationControl
+                icon={faLaughBeam}
+                title="I really love this!"
+              />
```

Comment:
I think we can use the html emoticons that I shared, then we won't need much styling